plausible/exports.sh

```diff
@@ -0,0 +1,2 @@
+export APP_PLAUSIBLE_SECRET_KEY="$(openssl rand -base64 48)"
+export APP_PLAUSIBLE_VAULT_KEY="$(openssl rand -base64 32)"
```

Comment:
Because these are in an `exports.sh` file, they will be regenerated fresh every time the Plausible app is restarted (through installation, update, or restart).

@arguser do you happen to know whether this behaviour is fine in Plausible? For example, if these are used to encrypt data, then on app restart a new key would be generated that potentially could not decrypt existing data.

Reply:
@nmfretz This is what I found:

> SECRET_KEY_BASE
> Configures the secret used for sessions in the dashboard, doesn't have any defaults and needs to be provided in the ENV vars, can be generated with OpenSSL

> TOTP_VAULT_KEY
> Configures the secret used for encrypting TOTP secrets at rest using AES256-GCM, doesn't have any defaults and needs to be provided in the ENV vars, can be generated with OpenSSL

I suspect that we need to have the same key on restart, specially for TOTP.

Reply:
@nmfretz what alternatives do we have to solve this situation?

Reply:
Thanks for looking that up @arguser. I've gone ahead and modified these so that they are unique for every user, but are deterministically derived such that they stay the same across app restarts: https://github.com/getumbrel/umbrel-apps/pull/1371/commits/0456dd1b2603ad5d98bb22ba6bb055a31c10521e

`SECRET_KEY_BASE` can be any string, so we are using ${APP_SEED} which is a unique 256 bit long hex string deterministically derived from a user's Umbrel seed and an app's ID.

`TOTP_VAULT_KEY` MUST be 32 bytes long, so we're using openssl but making it deterministic by utilizing this derive_entropy function from the apps ecosystem on umbrelOS:
https://github.com/getumbrel/umbrel/blob/7103e98909041bb8e15702b4b2e3e7f78da4b351/packages/umbreld/source/modules/apps/legacy-compat/app-script#L69-L88

```
export APP_PLAUSIBLE_VAULT_KEY=$(openssl dgst -sha256 -binary <<< "$(derive_entropy "env-${app_entropy_identifier}-APP_VAULT_KEY")" | base64)
```